test/string.ts

```diff
@@ -163,6 +163,16 @@ test('string.empty', t => {
 	}, {message: 'Expected string to be empty, got `foo`'});
 });
 
+test('string.nonBlank', t => {
+	t.notThrows(() => {
+		ow('foo', ow.string.nonBlank);
+	});
+
+	t.throws(() => {
+		ow(' ' as any, ow.string.nonBlank);
```

Comment:
I don't know what the point of the `as any` is here, I just copied the existing test...

Reply:
I think it was used to silence some TS error.

Reply:
Shall I boyscout the other instances of it away too, or only in the new code?

Reply:
Cleanups are better done in a separate PR.